HealthcareSystem/Backend/Repository/FeedbackRepository/MySqlFeedbackRepository/MySqlFeedbackRepository.cs

```diff
@@ -18,27 +18,45 @@ public MySqlFeedbackRepository(MyDbContext context)
         {
             _context = context;
         }
+        /// <summary>
+        /// adding new feedback to database
+        /// </summary>
+        /// <param name="feedback">an object to be added to the database</param>
```

Comment:
I am confused now, are you going to use this:

```sh
        /// <summary>
        /// **description** 
        /// </summary>
```

or this:

```sh
        /// <summary>
        /// / **description** 
        /// </summary>
```

@jelenabudisa

Reply:
Second alignment representation is automatically genereted by Visual Studio, furthermore i shall use the mentioned version. Additionally, in the next commit it will be used in that form above all methods.

Reply:
Alright, in the next commit just use one convention. And we will finish with this branch. You make a good job, well done! 

Could you suggest your team writing documentation while coding so that we have a better explanation of what a class or method is used for in the future?

Reply:
Thank you! Will do.